gradle/versions.gradle

```diff
@@ -77,6 +77,7 @@ ext {
     jcipAnnotationVersion = "1.0-1"
     openTelemetryVersion = "1.18.0"
     jetpackDataStoreVersion = "1.0.0"
+    blockstoreVersion="16.2.0"
```

Comment:
>blockstoreVersion="16.2.0" [](http://example.com/codeflow?start=4&length=26)

it it enough to add just here or do you need it to be added in android-complete too?

Reply:
I am having some issues with my local android-complete project. Hence the delay in raising that PR. But yes, it is needed in android-complete as well. Will send out an android-complete PR as soon as I resolve the issue on my local